bigquery/tables.js

```diff
@@ -15,308 +15,412 @@
-  return table.export(file)
+  // Exports data from the table into a Google Cloud Storage file
+  bigquery
+    .dataset(datasetId)
+    .table(tableId)
+    .export(storage.bucket(bucketName).file(filename))
     .then((results) => {
       job = results[0];
       console.log(`Job ${job.id} started.`);
       return job.promise();
     })
     .then((results) => {
       console.log(`Job ${job.id} completed.`);
-      return results;
+    })
+    .catch((err) => {
+      console.error('ERROR:', err);
     });
+  // [END bigquery_export_gcs]
 }
-// [END bigquery_export_gcs]
 
-// [START bigquery_insert_stream]
 function insertRowsAsStream (datasetId, tableId, rows, projectId) {
+  // [START bigquery_insert_stream]
+  // Imports the Google Cloud client library
+  const BigQuery = require('@google-cloud/bigquery');
+
+  // The project ID to use, e.g. "your-project-id"
+  // const projectId = "your-project-id";
+
+  // The ID of the dataset of the table into which data should be inserted, e.g. "my_dataset"
+  // const datasetId = "my_dataset";
+
+  // The ID of the table into which data should be inserted, e.g. "my_table"
+  // const tableId = "my_table";
+
   // Instantiates a client
   const bigquery = BigQuery({
     projectId: projectId
   });
 
-  // References an existing dataset, e.g. "my_dataset"
-  const dataset = bigquery.dataset(datasetId);
-  // References an existing table, e.g. "my_table"
-  const table = dataset.table(tableId);
-
   // Inserts data into a table
-  return table.insert(rows)
+  bigquery
+    .dataset(datasetId)
+    .table(tableId)
+    .insert(rows)
     .then((insertErrors) => {
       console.log('Inserted:');
       rows.forEach((row) => console.log(row));
-      return insertErrors;
+
+      if (insertErrors && insertErrors.length) {
+        console.log('Insert errors:');
+        insertErrors.forEach((err) => console.error(err));
+      }
+    })
+    .catch((err) => {
+      console.error('ERROR:', err);
     });
+  // [END bigquery_insert_stream]
 }
-// [END bigquery_insert_stream]
 
-// The command-line program
-const cli = require(`yargs`);
 const fs = require(`fs`);
 
-const program = module.exports = {
-  createTable: createTable,
-  listTables: listTables,
-  browseRows: browseRows,
-  deleteTable: deleteTable,
-  importLocalFile: importLocalFile,
-  importFileFromGCS: importFileFromGCS,
-  exportTableToGCS: exportTableToGCS,
-  insertRowsAsStream: insertRowsAsStream,
-  copyTable: copyTable,
-  main: (args) => {
-    // Run the command-line program
-    cli.help().strict().parse(args).argv; // eslint-disable-line
-  }
-};
-
-cli
+require(`yargs`) // eslint-disable-line
   .demand(1)
   .command(`create <datasetId> <tableId> <schema> [projectId]`, `Creates a new table.`, {}, (opts) => {
-    program.createTable(opts.datasetId, opts.tableId, opts.schema, opts.projectId || process.env.GCLOUD_PROJECT);
+    createTable(opts.datasetId, opts.tableId, opts.schema, opts.projectId || process.env.GCLOUD_PROJECT);
```

Comment:
Nit: is it possible to make a global `projectId` option with yargs that uses `process.env.GCLOUD_PROJECT` as a default? Might be fewer changes/less repetition...